src/sql/base/browser/ui/table/plugins/checkboxSelectColumn.plugin.ts

```diff
@@ -191,7 +191,8 @@ export class CheckboxSelectColumn<T extends Slick.SlickData> implements Slick.Pl
 			this._grid.render();
 		}
 
-		//Ensure that the focus stays correct
+		//Ensure that the focus stays on current selected checkbox row
```

Comment:
nit: isn't this making sure the focus is specifically on the checkbox's cell, not row?

Reply:
yeah it focus the cell! updated